vault/external_tests/identity/login_mfa_okta_test.go

```diff
@@ -6,20 +6,56 @@ package identity
 import (
 	"context"
 	"fmt"
+	"os"
 	"reflect"
+	"strings"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/helper/constants"
+	"github.com/hashicorp/vault/helper/testhelpers"
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
 	"github.com/hashicorp/vault/helper/testhelpers/minimal"
 )
 
-var (
-	org_name  = "<okta org name>"
-	api_token = "<okta api token>"
-)
+// To run these tests, set the following env variables:
+// VAULT_ACC=1
+// OKTA_ORG=dev-219337
+// OKTA_API_TOKEN=<generate via web UI, see Confluence for login details>
+// OKTA_USERNAME=test3@example.com
+// OKTA_PASSWORD=<find in 1password>
```

Comment:
I don't like the idea of editing the test source code just so I can run the tests. So I modified these tests to pull the relevant info from env vars instead.

I got this pattern from `builtin/credential/okta/backend_test.go`
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/builtin/credential/okta/backend_test.go#L25-L30

NB: I used my own Okta dev account to run these tests.

Reply:
Thanks for refactoring this! I agree that this makes more sense :)